### ISSUE TITLE
MCP: Screen reader fails to inform the user a tool was successfully run

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#247606]
Testing #247533

1. Tell the Agent to run a tool which requires user confirmation
2. Click `Continue`

🐛 Screen reader will read the chat response, but it will fail to acknowledge to the user that the tool was successfully run, unlike the visual UI which does have a green checkmark ✔ .

![Image](https://github.com/user-attachments/assets/7c43c40c-1cb4-40e4-8684-313940f21b2b)

**Comment by connor4312** [comment]
Imo this is good behavior. In a nice agentic mode I just want the agent to work and I only would want to know if tools fail. (Which we have partial representation of but I want to beef that up next iteration.) And the model tends to summarize the tool call result anyway.
